app/views/static/release_notes.html.erb

```diff
@@ -1,6 +1,20 @@
 <div class="container">
   <h1><strong>Poznámky k zmenám funkcionality a štruktúry dát</strong></h1>
   <div class="long-text">
+    <h2 class="anchorable">Zmena k 3. 12. 2020</h2>
+    <h3>Autoform API</h3>
+    <h4>Pridaný objekt <code>main_economic_activity</code></h4>
+    <ul>
+      <li>Pridaný atribút <code>code: integer</code></li>
+      <li>Pridaný atribút <code>name: string</code></li>
+    </ul>
+    <h4>Pridaný objekt <code>statutory_bodies</code></h4>
+    <ul>
+      <li>Štruktúra a dáta zhodné s objektom
+        <code>statutory</code>, ktorý je deprecated a v blízkej budúcnosti bude odstránený.
```

Comment:
@michal-rohacek toto by som preformuloval len na `<h4>Pridaný objekt <code>statutory_bodies</code>, ktorého atribúty sú zhodné s objektom `statutory`</h4>`, 
a odstranenie normalne pridaj to ako dalsiu zmenu do buducna, napr. `<h2 class="anchorable">Zmena k 1. 2. 2021</h2>` nech sa o tom vie uz teraz